kie-bom/pom.xml

```diff
@@ -2439,6 +2439,19 @@
         <type>test-jar</type>
       </dependency>
 
+      <dependency>
+        <groupId>org.kie.workbench.stunner</groupId>
+        <artifactId>kie-wb-common-stunner-kogito-runtime-common</artifactId>
```

Comment:
I spotted somewhere Roger asked not introduce this module, but reuse an existing one. 

Just checking if that was discussed more and this PR is correct or this PR will change.

Reply:
I'll remove it and move everything into the module Roger suggested, don't worry ;)

Reply:
@jomarko after talking to @romartin we won't remove this module for now, having it is a pain but looks like the best solution for now.